server/Dockerfile

```diff
@@ -1,15 +1,21 @@
+FROM kadalu/builder:latest as builder
+
+ENV PATH="/kadalu/bin:/opt/bin:/opt/sbin:$PATH"
+RUN python3 -m pip install glustercli
```

Comment:
This package was introduced when external Quotad was running gluster cli commands. Now this can be removed?

Reply:
We don't have control on having 'simple-quota' on External glusterfs. So, It may be needed with kubect-quotad.

About having it in server image, I am not sure!

Reply:
Ok, rechecked the code. We introduced glustercli in server image because we had quotad, which needed it. Now that its not there, I feel we can remove it.